### ISSUE TITLE
No error-handling required?

### ISSUE THREAD

**Issue by ORESoftware** [saadtazi/firefox-profile-js#85]
Very curious to see if you know why no error-handling would be required here:

```js
  let driverPromise = new Promise(function(resolve){

    var capabilities = webdriver.Capabilities.firefox();

    fp.encoded(function(encodedProfile) {
      capabilities.set('firefox_profile', encodedProfile);

      var wd = new webdriver.Builder().
      withCapabilities(capabilities).
      build();

      resolve(wd);
    });

  });
```

surely an error could be raised somewhere here, how could I handle it?

The reason I bring the issue up here, is so that your Readme could be updated accordingly (or examples, whatever).

thanks!

**Comment by saadtazi** [comment]
Hi,
Are you referring to `encoded()` which is callback based and its callback doesn't take `error` as its first parameter? What i am trting ro say: the api should definitively be `encoded(function (err,profile) {...})`. 
I'll see if i have time to release a new version rhis week-end (breaking change...)... PR more than welcome!

**Comment by ORESoftware** [comment]
Yeah, I am surprised that it's not an error-first callback

**Comment by saadtazi** [comment]
Fixed in version 1.0.0.
